_layouts/front.html

```diff
@@ -98,18 +98,49 @@ <h3>Mentorship</h3>
                 <h2 class="section-heading">Get Involved!</h2>
                 <hr class="dark">
                 <p class="text-muted">
-                    We are always looking for high school and college students to help us support our community! 
-                    Whether you are good at art, like to post on social media, have great scores on tests and in school, 
-                    want to share your college application journey with others, or like writing articles, 
+                    We are always looking for high school and college students to help us support our community!
+                    Whether you are good at art, like to post on social media, have great scores on tests and in school,
+                    want to share your college application journey with others, or like writing articles,
                     there's always a position for you on our team!</p>
                 <a href="/get-involved" class="btn btn-primary btn-xl">Learn More!</a>
             </div>
         </div>
     </div>
   </section>
 
+  <section class="bg-light" id="footer">
+    <div class="container">
+      <div class="row">
+        <div class="col-sm-3">
+          <h3 class="section-heading">Social Media</h3>
+          <ul class="list-unstyled" id="footer-links">
+            <li><a href="https://discord.myeduvisa.org">Discord</a></li>
+            <li><a href="https://www.instagram.com/myeduvisa/">Instagram</a></li>
+            <li><a href="https://www.facebook.com/teameduvisa/">Facebook</a></li>
+            <li><a href="https://www.youtube.com/channel/UC6KGOBlBVm8yp172XAoNKdQ">YouTube</a></li>
+            <li><a href="https://www.linkedin.com/company/myeduvisa/">LinkedIn</a></li>
+            <li><a href="https://twitter.com/myeduvisa">Twitter</a></li>
+            <li><a href="https://www.reddit.com/user/eduvisa/">Reddit</a></li>
+            <li><a href="https://linktr.ee/eduvisa">Linktree</a></li>
+          </ul>
+        </div>
+
+        <div class="col-sm-3">
+          <h3 class="section-heading">Join Us</h3>
+          <ul class="list-unstyled" id="footer-links">
+            <li><a href="/get-involved"> Get Involved </a></li>
+            <li><a href="/get-involved/tutoring"> Tutoring </a></li>
+            <li><a href="/get-involved/mentorship"> Mentorship </a></li>
+            <li><a href="/get-involved/specializations"> Specializations </a></li>
+            <li><a href="mailto:team@myeduvisa.org">Contact Us</a></li>
+            <li><a href="https://donate.myeduvisa.org">Donate</a></li>
+          </ul>
```

Comment:
I think these names could be more descriptive--for example: 
"Tutoring" -> "Become a Tutor"
"Mentorship" -> "Become a Mentor"
"Contact Us" -> "Email Us" (so that it's clear the link is a `mailto:`, my first impression was that it went to a form)

Specializations is a bit harder, maybe "Join our other departments"?

Reply:
Alternatively, change the "Contact Us" to something standalone like SimplyNeuroscience, which I think would help break up the text-only columns look. 
![image](https://user-images.githubusercontent.com/29838112/97510744-c8696200-195b-11eb-9493-f154f445f3a8.png)
(I did some research, I think it's fine just to put team@myeduvisa.org as the text of the `mailto:`).  We can also write something along the lines of "(copyright) 2020 by The EduVisa Community, a 501(c)(3) nonprofit organization", or we could put that as a standalone thing at the bottom like EachMind 
![image](https://user-images.githubusercontent.com/29838112/97511236-fb602580-195c-11eb-93ff-20252b23bb33.png)

Reply:
Yeah I agree with the first comment. You can put "Become a Specialist" for specializations.

And let's make the copyright thing an individual block like EachMind and put Email Us instead of Contact

Reply:
@aditinyn Ahhh I hate "Become a Specialist", it tells you absolutely nothing (wtf is a "specialist"??). Maybe we can mirror the language on the `/get-involved` page and write it as "Explore Our Other Positions"?